docs/book/v2/save-handler.md

```diff
@@ -5,36 +5,6 @@ are decoupled from PHP's save handler functions and are only implemented as a
 PHP save handler when utilized in conjunction with
 `Laminas\Session\SessionManager`.
 
-## Cache
-
-`Laminas\Session\SaveHandler\Cache` allows you to provide an instance of
-`Laminas\Cache\Storage\Adapter\AdapterInterface` to be utilized as a session save
-handler. Generally if you are utilizing the `Cache` save handler; you are likely
-using products such as memcached.
-
-### Basic usage
-
-A basic example is one like the following:
-
-```php
-use Laminas\Cache\StorageFactory;
-use Laminas\Session\SaveHandler\Cache;
-use Laminas\Session\SessionManager;
-
-$cache = StorageFactory::factory([
-    'adapter' => [
-       'name' => 'memcached',
-       'options' => [
-           'server' => '127.0.0.1',
-       ],
-    ],
-]);
-
-$saveHandler = new Cache($cache);
-$manager = new SessionManager();
-$manager->setSaveHandler($saveHandler);
-```
-
```

Comment:
This needs to be reverted. The v2 docs should remain unchanged, or, amended to include a note such as

```markdown
CAUTION: **Some Headline**
In version 3, `laminas-session` will no longer support `laminas-cache` as a storage backend.
Instead, a PSR-16 cache will be required.
You can find out more in the [migration guide](../v3/migration.md).
```

To be clear, just reverting the deletion is fine IMO, because the migration guide should have all the relevant info.